src/main/resources/config.yml

```diff
@@ -9,3 +9,6 @@ redis:
 givecreativeplot:
   max-plots: 9
   min-plots: 4
+extras:
+  skincontrol: true
+  delayedcommands: true
```

Comment:
Remove these from `extras` and put them outside along with `redis` and `givecreativeplot` and do necessary changes in the code to adjust.